packages/flutter_tools/templates/plugin/android-kotlin.tmpl/build.gradle.tmpl

```diff
@@ -2,7 +2,7 @@ group '{{androidIdentifier}}'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.3.72'
```

Comment:
It might worth checking with the Gradle team to confirm how this affects subprojects with different Kotlin version. Particularly, they suggested to use the composite build flow to isolate dependencies.